pages/article/ArticleHandler.inc.php

```diff
@@ -17,18 +17,47 @@
 import('classes.handler.Handler');
 
 class ArticleHandler extends Handler {
-	/** journal associated with the request **/
+	
+	/** @var journal object: journal associated with the request **/
 	var $journal;
 
-	/** issue associated with the request **/
+	/** @var issue object: issue associated with the request **/
 	var $issue;
 
-	/** article associated with the request **/
+	/** @var article object: article associated with the request **/
 	var $article;
 
-	/** galley associated with the request **/
+	/** @var galley object: galley associated with the request **/
 	var $galley;
-
+	
+	/** @var int: type of the requested object (article, galley or file) **/
+	var $type;
+	
+	/** @var int: article ID associated with the request **/
+	var $articleId;
```

Comment:
But then it has to be ensured at the appropriate places in the file that both are consistent. For example, article object is given as parameter to the hooks, for plugins to do something with it, so it should be consistent with the actual articleId. Also sometimes $this->article->getId() and sometimes $this->articleId is used.